page-objects/pages/coursePages/CRSSCourseListPage.js

```diff
@@ -11,6 +14,7 @@ const searchCourseFiled = ".input-group .search-field";
 const courseWrapper = ".sc-card-wrapper";
 const titleOfCourse = ".title";
 const memberBtn = ".btn-member";
+const homeworktab = '.tabs button[data-testid="hometasks"]';
```

Comment:
I think that it is always better to keep selectors together as one object, like const selectors= {selector1, selector2...}. and the add selectors to module.exports. What is your opinion? as I did refactoring I tried to keep it everywhere  like that.

Reply:
done